scene/animation/animation_player.h

```diff
@@ -176,7 +178,9 @@ class AnimationPlayer : public AnimationMixer {
 	Tween::EaseType get_auto_capture_ease_type() const;
 
 	void play(const StringName &p_name = StringName(), double p_custom_blend = -1, float p_custom_scale = 1.0, bool p_from_end = false);
+	void play_section(const StringName &p_name = StringName(), const StringName &p_start_marker = StringName(), const StringName &p_end_marker = StringName(), double p_custom_blend = -1, float p_custom_scale = 1.0, bool p_from_end = false);
```

Comment:
It would be nice for scripting/future extensions to not have to rely on markers to play a section. In other words, having a play method that just takes two `double`s for the start and end (AFAIK there wasn't something like this before). Maybe have `play_section` take the raw start and end values and add a `play_region` method that does what this does?

Not sure which is the right naming though, both "region" and "section" can refer to either an arbitrary slice or one delineated by markers...

Reply:
Since it plays only a part of the animation, I think both region and section have the same meaning, but I don't know which one feels more natural.

Personally, I feel that regions are meant to be more of a geographic zone, so I just recommend sections. However, we are not an English speaker, so I don't know which is better for native English.

Reply:
Well in this context of timelines, "region", "section" and "AB" are used pretty much interchangeably. I have a slight preference for "region", but if you want to stick with "section", how about a `play_section` method that takes start/end values in seconds, and a `play_marker_section` method that takes the names of the start and end marker?

Reply:
I have a suggestion too if you don't mind. How about a "clip"? So `play_clip` would take start/end times, and `play_section` would take start/end markers. Clips are more often associated with timestamps from what I've seen.

Reply:
Animation clip is not recommended as it exists as a separate feature during import

Reply:
clip and tracks are already used in other ways in animation.

Reply:
Going with @lapspider45 's suggestion of `play_section` and `play_marker_section` for now. I can change the names again if needed later.

Reply:
For consistency with other APIs, I recommend `play_section_with_markers()`.